jina/serve/runtimes/gateway/websocket/app.py

```diff
@@ -130,6 +130,18 @@ async def send(self, websocket: WebSocket, data: DataRequest) -> None:
 
     streamer.Call = streamer.stream
 
+    @app.get(
```

Comment:
I do not understand, how is it different than the `@app.websocket('/')`?

Reply:
FastAPI handles http / ws endpoints differently and redirects traffic according to the request type. Hence both `/` endpoints would be treated differently. 

AWS's ALB cannot send websocket-based requests to Gateway. Hence we configure ALB to send `GET /` request for healthcheck & accept `404` as a valid response code, which is wrong. This PR adds an actual `GET /` endpoint so ALB can get a 200 response.

![image](https://user-images.githubusercontent.com/9050737/170922094-f17826fc-8e5d-4998-b78a-15dd914ba82d.png)